pkg/gen/input/makefile/internal/file/makefile.go

```diff
@@ -132,6 +132,17 @@ build-docker: build-linux
 	@echo "====> $@"
 	docker build -t ${APPLICATION}:${VERSION} .
 
+.PHONY: lint-chart
+## lint-chart: runs ct against the default chart
+lint-chart:
+	@echo "====> $@"
+	rm -rf /tmp/$(APPLICATION)-test
+	mkdir -p /tmp/$(APPLICATION)-test/helm
+	cp -a ./helm/$(APPLICATION) /tmp/$(APPLICATION)-test/helm/
+	architect helm template --dir /tmp/$(APPLICATION)-test/helm/$(APPLICATION)
+	docker run -it --rm -v /tmp/$(APPLICATION)-test:/wd --workdir=/wd --name ct quay.io/giantswarm/helm-chart-testing:v3.0.0-rc.1 ct lint --validate-maintainers=false --charts="helm/$(APPLICATION)"
```

Comment:
what about setting the image and version in the variable section in line 31?

Reply:
Why do you think it's beneficial?

Reply:
In case of the image gets updated, was only a guess from my side that this happens at some point 😄. But no blocker from my side.

Reply:
I still want to drill it because I have my reasons to keep it here and I may change my mind if understand you better. Why do you think it's better to have it up in the line 31 section if the image gets updated?

My thinking was that if we remove the target we won't have any leftover variables. This is my rule of thumb when using the value once.

Reply:
The docker image there is not easy to see, so we might forget to update when needed. I thought it would be easier to see when it's in the vars section, so you don't have to scan through the lines.